tls/go-client/app/env.sh

```diff
@@ -0,0 +1,3 @@
+export CONF_CONSUMER_FILE_PATH=../profiles/dev/client.yml
```

Comment:
avoid use shell script to start. try if build/makefile work or not. pls. refer to HOWTO.md